tests/conftest.py

```diff
@@ -199,6 +199,24 @@ def nbrhosts(ansible_adhoc, testbed, creds):
         devices[k] = EosHost(ansible_adhoc, "VM%04d" % (vm_base + v['vm_offset']), creds['eos_login'], creds['eos_password'])
     return devices
 
+@pytest.fixture(scope="module")
+def fanouthosts(ansible_adhoc, creds):
+    """
+    Shortcut fixture for getting Fanout host
+    """
+    devices = {}
+
+    # TODO: Create a new module for get lab_graph
+    lab_graph = Parse_Lab_Graph(filename)
+    for device in lab_graph.devices:
+        if device['Type'] == 'FanoutLeaf' or device['Type'] == 'FanoutRoot':
+            # TODO: Get OS by device hwsku: https://github.com/Azure/sonic-mgmt/blob/11904a9cc64bfcd68910cbaa3843516ee3e7feae/ansible/testbed-new.yaml
+            os = 'eos'
```

Comment:
Here is a TODO. I need to get the os type by device hwsku.
I can found the mapping in the linked file, not sure if this is the right way to get the os type?

Reply:
Can you get OS type from "show version"?

Reply:
I don't think 'show version' work because this is a n 'checken-and-egg' problem. We need to get os type to login and then we can 'show version'.